src/TextTagger.php

```diff
@@ -0,0 +1,135 @@
+<?php
+namespace Whatafix\TextTagger;
+use Whatafix\TextTagger\Contracts\TextTaggerInterface;
+class TextTagger implements TextTaggerInterface
+{
+    const TAG_LIST = [
+        'family'=>[
+            'fils',
+            'fille',
+            'parent',
+            'mère',
+            'père',
+            'maman',
+            'papa',
+            'grand-parent',
+            'grand-mère',
+            'grand-père',
+            'tante',
+            'oncle',
+            'neveu',
+            'nièce',
+            'cousin',
+            'cousine',
+            'épouse',
+            'mari',
+            'beau-frère',
+            'belle-soeur',
+            'frère',
+            'soeur',
+            'enfant'
+        ],
+        'walk'=>[
+            'ballade',
+            'sortie',
+            'promener',
+            'promenade'
+        ],
+        'bathroom'=>[
+            'lavabo',
+            'baignoire',
+            'douche',
+            'toilette',
+            'savon',
+            'shampoing',
+            'pharmacie',
+            'porte-serviettes',
+            'bain'
+        ],
+        'school'=>[
+            'école',
+            'faute',
+            'grammaire',
+            'éducation',
+            'punition',
+            'bulletin',
+            'institutrice',
+            'étude',
+            'problème',
+            'cours',
+            'classe',
+            'devoir',
+            'élève',
+            'instituteur',
+            'leçon',
+            'savoir',
+            'science',
+            'apprentissage',
+            'cahier',
+            'mot',
+            'écolier',
+            'parole',
+            'récréation',
+            'rentrée',
+            'vacances',
+            'exemple',
+            'instruction',
+            'livre',
+            'page',
+            'stylo',
+            'math',
+            'physique',
+            'chimie',
+            'collège',
+            'lycée',
+            'formation',
+            'scolaire',
+            'enseignement',
+            'correction',
+            'poésie',
+            'poème',
+            'dictionnaire',
+            'rédaction',
+            'note',
+            'question',
+            'examen',
+            'oral',
+            'travail'
+        ],
+        'train'=>[
+            'rail'
+        ]
+    ];
+
+    /**
+     * Return the tags for a specific text
+     * @param string $text
+     * @return string[]
+     */
+    public function getTags(string $text): array
+    {
+        $tags = array();

```

Comment:
:+1: Bien si le foreach ne ressort pas de tag on aura un array vide, parfait.

Préfère une short syntax pour les array :

```suggestion
        $tags = [];

```